istio/istio.spec

```diff
@@ -14,7 +14,7 @@
 %global debug_package   %{nil}
 %endif
 
-%global git_commit c5db59a9076a5b7bf968ae8f69cc786b3e015d78
+%global git_commit fd5254d87df6733d09972d53d532564d6c499c2b
```

Comment:
This is wrong. It should be `7d3618ce62aed95e4d1d064af6d237d3311aff30`.

Reply:
Good catch. Thanks. I just pushed that to product. let me fix it.

Reply:
Interesting. I had the right one there, but wrong here.